frame/fast-unstake/src/lib.rs

```diff
@@ -113,10 +113,12 @@ pub mod pallet {
 			+ IsType<<Self as frame_system::Config>::RuntimeEvent>
 			+ TryInto<Event<Self>>;
 
-		/// The amount of balance slashed per each era that was wastefully checked.
-		///
-		/// A reasonable value could be `runtime_weight_to_fee(weight_per_era_check)`.
-		type SlashPerEra: Get<BalanceOf<Self>>;
+		/// The currency used for deposits.
+		type DepositCurrency: ReservableCurrency<Self::AccountId, Balance = Self::CurrencyBalance>;
```

Comment:
I don't see `CurrencyBalance` defined here.

Reply:
It's inherited from `pallet_staking`

Reply:
I'd use `BalanceOf<Self>` to avoid confusion, `BalanceOf<Self>` is statically the same as `Self::CurrencyBalance`.